test/unit/lib/present_page_to_publishing_api_test.rb

```diff
@@ -0,0 +1,17 @@
+require "test_helper"
+
+class PresentPageToPublishingApiTest < ActiveSupport::TestCase
```

Comment:
We were wondering why this is in a separate file to the test/unit/present_page_to_publishing_api_test.rb tests. As they relate to the same class.